test/run_unit_tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
This shouldn't have been committed; instead change run_unit_tests.sh.in and commit that?  This file should be written in our build dir by configure, not in our source dir manually.